retention/src/test/java/org/apache/falcon/retention/FeedEvictorTest.java

```diff
@@ -462,6 +462,43 @@ public void testFeedBasePathExists() throws Exception {
         }
     }
 
+    @Test
+    public void testEvictionStatsMetaWithNoPattern() throws Exception {
+        try {
+            Configuration conf = cluster.getConf();
+            FileSystem fs = FileSystem.get(conf);
+            fs.delete(new Path("/"), true);
```

Comment:
@bvellanki, see the pom.xml that the tests are *not* executed in parallel. The change to make the tests dependent on one another is superfluous. 

          <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-surefire-plugin</artifactId>
                <version>2.16</version>
                <configuration>
                    <redirectTestOutputToFile>true</redirectTestOutputToFile>
                    <forkCount>1</forkCount>
                    <reuseForks>false</reuseForks>
                    <forkMode>always</forkMode>
                    <argLine>-Djava.awt.headless=true -Djava.security.krb5.realm= -Djava.security.krb5.kdc=</argLine>
                    <excludedGroups>${excluded.test.groups}</excludedGroups>
                </configuration>
            </plugin>